portfolio/src/main/webapp/contact.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Nothings on this page. Maybe move your contact form here? Or delete it.

If you keep it,  copy/paste the same structure as your other pages (e.g. navigation bar, content box, etc) so that once you're on this page you can still navigate back, and it feel fluid jumping around the site.